### DISCUSSION TITLE
What is the schedule time to publish a release to crates.io with the large file read/write patch?

### DISCUSSION THREAD

**Discussion by fMeow** [Xuanwo/hdrs#84] [Q&A] [answered]
What is the schedule time to publish a release to crates.io with the large file read/write patch?

**Comment by Xuanwo** [comment]
now. (after all CI passed)

**Comment by fMeow** [comment]
Thanks a lot! :)

**Comment by Xuanwo** [comment, marked as answer]
https://github.com/Xuanwo/hdrs/discussions/87 has been released, PTAL @fMeow

**Comment by fMeow** [comment]
Much appreciated! :)

**Comment by fMeow** [comment]
I would like to ask what is the schedule time to publish a new patch release to crates.io? The the large file read/write bug is not fixed in v0.1.5, which actually shares the same bug with the previous version. So I don't think it necessary to yank v0.1.5 from crates.io

Also, we are in need of the new rename method to ensure reading completely uploaded files.

**Reply by Xuanwo** [reply]
I plan to release the new version later today~

**Reply by fMeow** [reply]
Thanks a lot!
